### ISSUE TITLE
[Bug]: Custom Chrome extension not load in electron app

### ISSUE THREAD

**Issue by ElectronMB** [electron/electron#42106]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

26.2.1

### What operating system are you using?

macOS

### Operating System Version

mac mini 14.4.1

### What arch are you using?

x64

### Expected Behavior

Open Chrome extension into the electron app.

### Actual Behavior

When I load an extension with the below document it throws the error in the terminal 

**Error:** 

  Manifest contains a differential_fingerprint key that will be overridden on the extension update.
  Permission 'scripting' is unknown or URL pattern is malformed.
  
 **Document I used:**

--> https://www.electronjs.org/docs/latest/tutorial/devtools-extension#:~:text=To%20load%20an%20extension%20in,the%20extension%20in%20Google%20Chrome

--> https://github.com/samuelmaddock/electron-browser-shell

I take all the above link references but I get the same error in every solution.

### Testcase Gist URL

https://github.com/samuelmaddock/electron-browser-shell

### Additional Information

Basically, I have my own unpack extension, or also it's in the web store and I want to add that extension to my electron app.

Shortly I want to use my electron app like Google Chrome browser in which I can add extensions and parallel run the electron pages as well but not able to do so with above given reference

**My extension manifest version is 3**

My electron version is : 26.2.1

**Comment by codebytere** [comment]
Thanks for reaching out!

Because we treat our issues list as the team's backlog, we close issues that are questions since they don't represent a task needing to be completed. This forum is for raising bugs or requesting new features, and for most questions about Electron there are a lot of more appropriate [community options](https://github.com/electron/electron#community). There are helpful people in this [Discord](https://discord.gg/electronjs) or the [subreddit](https://www.reddit.com/r/electronjs) that should be willing to point you in the right direction.
